docs/getting-started/linux.rst

```diff
@@ -5,7 +5,7 @@ This walkthrough will create a simple console application using ASP.NET 5 and
 the SQLite provider.
 
 .. note::
-    This article was written for beta 8 of ASP.NET and EF7.
+    This article was written for RC1 of ASP.NET and EF7.
 
     You can find nightly builds of the EF7 code base hosted on https://www.myget.org/gallery/aspnetvnext/ but the code base is rapidly changing and we do not maintain up-to-date documentation for getting started.
 
```

Comment:
Can you swap this out for `.. include:: /rc1-notice.txt` and put it as the first thing in the file (trying to standardize on that across all docs)